kubehub/k8s_deploy/kubespray_deploy.py

```diff
@@ -0,0 +1,87 @@
+from re import findall
+from subprocess import Popen
+from django.forms.models import model_to_dict
+
+from ..models.vm_group import VM
+from ..models.k8s_cluster import KubernetesCluster
+from ..models.kubespray_deploy import KubesprayDeploy
+from ..k8s_deploy.deploy_log_file_create import create_log_file
+from ..k8s_deploy.deploy_log_directory_create import create_deploy_logs_dir
+from ..serializers.k8s_cluster_serializer import KubernetesClusterSerializer
+from ..serializers.kubespray_deploy_serializer import KubesprayDeploySerializer
+
+
+def kubespray_deploy(k8s_cluster_id):
+    log_dir = create_deploy_logs_dir()
+    k8s_cluster_id = KubernetesCluster.objects.get(pk=k8s_cluster_id).id
+    vm_group_id = KubernetesCluster.objects.get(pk=k8s_cluster_id).vm_group.id
+    vm_group__instance = VM.objects.filter(vm_group=vm_group_id).values_list('ip', flat=True)
+    vms_ip = list(vm_group__instance)
+    nomber_of_node = len(vms_ip) + 1
+    virtual_machine_ip = (" ".join(vms_ip))
+    cmd = ["./scripts/cluster_create.sh", virtual_machine_ip]
+    kubespray_deploy_data = {
+        'status': "deploying",
+        'vm_group': vm_group_id,
+        'k8s_cluster': k8s_cluster_id
+    }
+    k8s_cluster_status_update(
+        pk=k8s_cluster_id,
```

Comment:
don't user pk as variable name, commonly use practice is id

Reply:
fixed in 17ce6ec id used as variable name instead of pk